src/System.Data.SqlClient/tests/project.json

```diff
@@ -0,0 +1,14 @@
+﻿{
+    "dependencies": {
+        "Microsoft.CSharp": "4.0.1-rc3-23722",
```

Comment:
@nadiatk There are build breaks because of compilation errors. 
The message is project.json package prerelease is 'rc3-23722', but expected 'rc3-23725'
You need to update the package versions for the Project.json packages.